### ISSUE TITLE
Axes Labes "image.plot(..)"

### ISSUE THREAD

**Issue by mgrove-wwu** [hyperspy/hyperspy#1926]
Hello together!
I am currently starting to work with hyperspy starting my master thesis. 
I tried to use the .plot() routine to visualize diffraction patterns/ high resolution images.
Sadly, i ran into trouble changing the axes labels to my preferences.
The name can be easily changed through the gui or just calling the .name attribute, though the axes labels will be embedded with the word "axis" at the end. I will get this result:
"x axis"
x - the name i was specifying
I cant get hyperspy to dismiss the extra word "axis" and it really annoys me :/ especially because my plots need to be in german also for presentational needs.
I would be glad if you guys can give me a hint, because i went through the documation trying to find anything, but wasnt successful.

**Comment by ericpre** [comment]
You have several options:
1. Use the `hs.plot.plot_images` function:
```python
import hyperspy.api as hs
s = hs.datasets.example_signals.object_hologram()
axes = hs.plot.plot_images(s, axes_decor='off')
# by default a list of axis is return
ax = axes[0]
```
`hs.plot.plot_images` returns an matplotlib axis object (here `ax`) which you can use to tweak your plot.

2. tweak the plot by accessing the matplotlib axes (or the matplotlib figure) of your object:

```python
import hyperspy.api as hs
s = hs.datasets.example_signals.object_hologram()
s.plot()
ax = s._plot.signal_plot.ax
ax.set_xlabel('your label in german')
```

**Comment by mgrove-wwu** [comment]
Thank you for the really fast answer.
I tried .plot_images before, but axes_decor was not working as intended. It didnt change anything. I will try setting the debug information to get more information, but for now the matplotlib option works well and i am already used to working with the package.
Many thanks again!
Scincerely Maximilian!

**Comment by magnunor** [comment]
If you're going the matplotlib way, there is a Jupyter notebook explaining the basics of using HyperSpy signals with matplotlib: https://github.com/hyperspy/hyperspy-demos/blob/master/making_figures/making_signal2d_figure_using_matplotlib.ipynb

**Comment by ericpre** [comment]
The issue with the `hs.plot.plot_images` was fixed in #1914. If you install the last hyperspy release, this should work. If you are using the hyperspy bundle, it is better to uninstall and reinstall the bundle for this release.

**Comment by mgrove-wwu** [comment]
@ericpre 
Yep, that is the issue it seems.

I will just adjust the conda environment.

Sorry for being a confused student and thanks for the time!

**Comment by ericpre** [comment]
Closing this issue since this is fixed in the last release (1.3.1).
